src/BreadcrumbLink.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+final class BreadcrumbLink
```

Comment:
Looks like a general link, not a breadcrumb-specific one.

```suggestion
final class Link
```

Reply:
done.